transferui/src/main/java/com/hyperwallet/android/ui/transfer/viewmodel/CreateTransferViewModel.java

```diff
@@ -71,6 +72,9 @@
     private static final String DESTINATION_AMOUNT_INPUT_FIELD = "destinationAmount";
     private static final String DESTINATION_TOKEN_INPUT_FIELD = "destinationToken";
     private static final String PRIVATE_TOKEN_PREFIX = "trm-";
+    public static final String CURRENCY_DOT_SEPARATOR = ".";
+    public static final String REGEX_ONLY_NUMBER_AND_DECIMAL = "[^0-9.]";
```

Comment:
I don't see any usages for `REGEX_ONLY_NUMBER_AND_DECIMAL`

Reply:
removed